Deploy

WanderingStar opened the pull request:
Changes for deployment on Linode:
- Bind postgres to 5433 instead of 5432 to avoid collision with other DB running there
- Use gkg_idetect table instead of Document
- Run as uid 1001, not 1000 because that's what matches idetect on the server
- frozen requirements.txt and docker pre-install
- tune number of workers

WanderingStar committed:
fixes for deployment on linode

WanderingStar committed:
fix db_url, replace $UID with 1001

WanderingStar committed:
fix more imports

WanderingStar committed:
tuning

simonb83 committed:
Update setup script for new models and add script for loading urls

simonb83 committed:
fix encoding error

WanderingStar committed:
load models in setup.py

WanderingStar committed:
frozen packages

WanderingStar committed:
replace Document with Gkg

WanderingStar committed:
wrap api routes that use sessions with try/finally

WanderingStar committed:
worker counts 16/3/3

WanderingStar committed:
port 5432

WanderingStar committed:
pin gensim==2.3.0

simonb83 committed:
change config

simonb83 committed:
change config

simonb83 committed:
Wrap mapzen call with try...except to deal with errors

simonb83 committed:
Wrap mapzen call with try...except to deal with errors

simonb83 committed:
Remove call to common_names function

simonb83 committed:
Remove call to common_names function

simonb83 committed:
Return identified country code from pycountry

simonb83 committed:
Return identified country code from pycountry

simonb83 committed:
Add unique constraint to location_name

simonb83 committed:
Add unique constraint to location_name

simonb83 committed:
SQL for removing duplicates and adding unique constraint in database

simonb83 committed:
SQL for removing duplicates and adding unique constraint in database

simonb83 committed:
Remove geotagging part from fact extraction

simonb83 committed:
Remove geotagging part from fact extraction

simonb83 committed:
Add methods to geotagger for processing locations for a given article

simonb83 committed:
Add methods to geotagger for processing locations for a given article

simonb83 committed:
Add new worker for running geotagging process

simonb83 committed:
Add new worker for running geotagging process

simonb83 committed:
unit tests for new geotagging functions / worker

simonb83 committed:
unit tests for new geotagging functions / worker

simonb83 committed:
Mirror deploy version of supervisor

WanderingStar committed:
don't use article.document_id in templates

WanderingStar committed:
remove imports of removed code

WanderingStar committed:
remove imports of removed code

WanderingStar committed:
use gkg_idetect table instead of gkg table

WanderingStar committed:
include date column

WanderingStar committed:
added some documentation

WanderingStar committed:
run setup.py before starting workers

simonb83 committed:
refugees v1

simonb83 committed:
Simplify SQL for dealing with duplicate locations

simonb83 committed:
Simplify SQL for dealing with duplicate locations

simonb83 committed:
Make coords_tostring function cleaner in geo_external

simonb83 committed:
Make coords_tostring function cleaner in geo_external

simonb83 committed:
Remove hard coded API key

simonb83 committed:
Remove hard coded API key

simonb83 committed:
SQL for adding new keywords

simonb83 commented:
All looks good to me...I think I might rebase my duplicate locations branch off this one to incorporate the gkg table into my unit tests.

simonb83 committed:
Ability to categorize refugee-related facts

WanderingStar committed:
Merge pull request #37 from simonb83/documentation

added some documentation

simonb83 committed:
update fact extractor tests

simonb83 committed:
Fix failing unit tests

simonb83 committed:
Add geotagger worker to config

simonb83 committed:
Remove reference to Document in api

simonb83 committed:
Add placeholder for Mapzen key in docker.env

simonb83 committed:
Deal with locations where unable to resolve country; ensure fact_locations unique; deal with minor errors in geotagging and API; add test to load data if necessary prior to launching fact extractor

simonb83 committed:
Add SQL for making fact-locations unique

simonb83 committed:
Add unique fact-location constraint in model.py

simonb83 committed:
Merging remote

simonb83 committed:
Correct config

simonb83 committed:
Correct refugee unit test

simonb83 committed:
remove unneccessary files

simonb83 committed:
Remove calls to print in interpreter.py

simonb83 committed:
Merge pull request #38 from simonb83/refugees

Refugees tracking as requested by milanoleonardo

simonb83 committed:
Add notebook for deleting facts and changing article status

simonb83 committed:
Merge pull request #35 from simonb83/dup_locations

Deal with duplicate locations

simonb83 committed:
Switch from gkg_idetect to gkg table

simonb83 committed:
Scraping should fail when content is empty

simonb83 committed:
Scraping should fail if article not in Englilsh

simonb83 committed:
Commit after setting language and add exceptions to pdf scraping

simonb83 committed:
Merge pull request #43 from simonb83/article_status

Article status

milanoleonardo committed:
adding file to test external geocoders alternative to Mapzen

WanderingStar committed:
pin preinstalled packages

We're using pinned versions in requirements.txt, but we hadn't pinned
the same versions in the earlier places we installed those same packages.
This was causing spacy to be installed (1.10), then uninstalled, and
reinstalled (1.9)
